utils.ml

```diff
@@ -1,4 +1,21 @@
+module Json = struct
+  let clean_string s =
+    (* Remove backslashes and double quotes from the string *)
+    let buffer = Buffer.create (String.length s) in
+    String.iter
+      (fun c ->
+        match c with '\\' -> () | '"' -> () | _ -> Buffer.add_char buffer c)
+      s;
+    Buffer.contents buffer
+end
+
 module Email = struct
   let validate_email email =
-    match Emile.of_string email with Ok _ -> true | Error _ -> false
+    match Emile.of_string (Json.clean_string email) with
+    | Ok _ -> true
+    | Error s ->
+        Logs.err (fun m ->
+            m "Emile-Email-Validation %s"
```

Comment:
you can use "%a" directly here, and then Emile.pp_error -- no need for the asprintf (neither for the \n - which each log message has at the end)